scripts/plots/mesoscale/exevs_mesoscale_sref_cloud_plots.sh

```diff
@@ -201,6 +234,7 @@ else
    export err=$?; err_chk
 fi
 
+
```

Comment:
@ShelleyMelchior-NOAA Before this PR is merged, can you please also remove this extra space?